explore/src/main/scala/explore/common/ProposalQueries.scala

```diff
@@ -114,9 +114,7 @@ trait ProposalQueries:
     def toInput: ProposalPropertiesInput =
       ProposalPropertiesInput(
         callId = proposal.callId.orUnassign,
-        title = proposal.title.orUnassign,
         category = proposal.category.orUnassign,
-        `abstract` = proposal.abstrakt.orUnassign,
```

Comment:
no abstract?

Reply:
The program description is now used as the abstract. That way non-science programs, which don't have proposals, can have a description field, too.